index.html

```diff
@@ -1,76 +1,200 @@
+        <h3>
+          <a href="/sdks/android/"><i class="fa fa-android"></i> Android SDK</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
         <p>
-          <a href="http://www.feedhenry.com/feedhenry-and-red-hat-pushing-ahead-with-integrations/"><img src="{{ site.baseurl }}/img/logos_home/feedhenry.gif" alt="feedhenry"  /></a>
-          <a href="https://developers.openshift.com/en/xpaas-unified-push.html"><img src="{{ site.baseurl }}/img/logos_home/openshift.gif" alt="openshift"  /></a>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
         </p>
-      </div><!-- container -->
-    </section>
\ No newline at end of file
+        <p>
+          <a href="/sdks/android/" class="btn btn-primary">Learn More</a>
+        </p>
+      </div>
+
+      <div class="col-md-3">
+        <h3>
+          <a href="/sdks/cordova/"><i class="font-icon icon-cordova"></i> Cordova SDK</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
+        <p>
+          <a href="/sdks/cordova/" class="btn btn-primary">Learn More</a>
+        </p>
+      </div>
+
+      <div class="col-md-3">
+        <h3>
+          <a href="/sdks/ios/"><i class="fa fa-apple"></i> iOS SDK</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
+        <p>
+          <a href="/sdks/ios/" class="btn btn-primary">Learn More</a>
+        </p>
+      </div>
+
+      <div class="col-md-3">
+        <h3>
+          <a href="/sdks/xamarin/"><i class="font-icon icon-xamarin"></i> Xamarin SDK</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
+        <p>
+          <a href="/sdks/xamarin/" class="btn btn-primary">Learn More</a>
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="custommers">
+  <div class="container">
+    <h2 class="section-header">Mobile development begins here.</h2>
+    <p>Using mobile technologies will never be so easy as in the cloud. AeroGear supports several cloud-enabled providers and
+      technologies.</p>
+    <p>
+      <a target="_blank" href="http://www.feedhenry.com/feedhenry-and-red-hat-pushing-ahead-with-integrations/">
```

Comment:
Do we still want to point to feedhenry? also the link redirects to redhat.com.

Reply:
+1, I'd say yank refs to feedhenry in general

Reply:
Good point :+1:

Reply:
This whole section actually seems outdated. Is there anything this should be replaced with? @jasonmadigan @laurafitzgerald